notebooks/glossis-currents.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
This is just here for reference and not used right?

Reply:
This is  a notebook to try things out.